packages/components/psammead-navigation/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 9.2.13 | [PR#4587](https://github.com/bbc/psammead/pull/4587) Fix a11y spans and comma bugs |
```

Comment:
Should we be more specific? Fix for TalkBack incorrectly reading links in nested span? And does this PR fix the comma bug?

Reply:
No it does not fix the comma bug, I need to update these PRs 😅

Reply:
And yes, will try to be more specific